osx/make-installer.sh

```diff
@@ -205,9 +177,9 @@ fi
 
 # Sparkle settings, see https://sparkle-project.org/documentation/customization/#infoplist-settings
 if [ -z "${ptb}" ]; then
-  /usr/libexec/PlistBuddy -c "Add SUFeedURL string https://feeds.dblsqd.com/MKMMR7HNSP65PquQQbiDIw/release/mac/x86_64/appcast" "${app}/Contents/Info.plist" || true
+  /usr/libexec/PlistBuddy -c "Add SUFeedURL string https://feeds.dblsqd.com/MKMMR7HNSP65PquQQbiDIw/release/mac/${ARCH}/appcast" "${app}/Contents/Info.plist" || true
```

Comment:
@vadi2 does this mean you have to do a **one-time** login to dblsqd and set up the "architectures" for `arm64` builds on the "release" and "public-test-builds" channels?